docs/tracking/google-tag-manager.md

```diff
@@ -1,17 +1,22 @@
 ---
-title: Google Tag Manager (GTM)
-description: Learn how to integrate Featurevisor SDK with Google Tag Manager
+title: Data Capturing via Google Tag Manager (GTM)
+description: Learn how to integrate Featurevisor SDK with Google Tag Manager and enable your experimentation reporting through custom dataLayer events
 ---
+## Intro:
 
-You can send Featurevisor activation events coming from SDK to Google Tag Manager. {% .lead %}
+Being able to report on the outcome of an experimnt by leveraging high-quality data is a key aspect of any business or individual who wants to take a step closer to achieving their goals and offering users an outstanding experience on their platform. This article guides you on how to run experimentation analysis by leveraging your existing frontend dataset and enabling segmentation for bahvioural analysis through an established data collection method: Google Tag Manager. 
 
-## Custom events in GTM
+## Guide:
 
-Before tracking, make sure you have custom events set up in your container.
+## Send an experiment activation event to the dataLayer and collect data via GTM
+
+The SDK integration section below provides a guide on how to push the `featureVisorActivation` event to the dataLayer created by Google Tag Manager. Once the event has been pushed to the dataLayer, the data can be sent to your Google Analytics property by firing a GA4 or Universal Analytics tag that leverages a Custom Event Trigger. Make sure to set the Custom Event Trigger to exactly match the name of the event pushed to the dataLayer (`featureVisorActivation`).
```

Comment:
`featurevisorActivation`.

Reply:
the word "dataLayer" can be wrapped in backticks so that it appears as code everywhere. 

like this:

```
This is a sentence with `dataLayer` in it.
```

This will then make the word render as code as seen here: `dataLayer`.

Reply:
done :)